.github/workflows/update-docs.yml

```
@@ -0,0 +1,63 @@
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v6
+        with:
+          token: ${{ env.ACTIONS_BOT_TOKEN }}
+          base: main
+          branch: auto-docs/update-rpcn-connector-docs
+          title: 'auto-docs: Update RPCN connector docs'
+          commit-message: 'docs: Update RPCN connector docs'
+          body: ${{ steps.generate.outputs.delta_report }}
+          labels: auto-docs
+          allow-empty: false
```

actionlint (1.7.7)

input "allow-empty" is not defined in action "peter-evans/create-pull-request@v6". available inputs are "add-paths", "assignees", "author", "base", "body", "body-path", "branch", "branch-suffix", "commit-message", "committer", "delete-branch", "draft", "git-token", "labels", "milestone", "path", "push-to-fork", "reviewers", "signoff", "team-reviewers", "title", "token"

(action)